tck/src/test/java/org/reactivestreams/tck/PublisherVerificationTest.java

```diff
@@ -717,6 +735,79 @@ public void cancel() {
   }
 
   /**
+   * Verification using a Publisher that supports multiple subscribers
+   * @param shouldBlowUp if true {@link RuntimeException} will be thrown during second subscription.
+   */
+  final PublisherVerification<Integer> multiSubscribersPublisherVerification(final boolean shouldBlowUp) {
+    return new PublisherVerification<Integer>(newTestEnvironment()) {
+
+      @Override
+      public Publisher<Integer> createPublisher(final long elements) {
+        return new Publisher<Integer>() {
```

Comment:
Would it make sense to factor this implementation out and running the TCK against it?